packages/strapi-parts/src/Link/Link.stories.mdx

```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
 
 # Link
 
-This is the doc of the `Link` component
+Links are used to send users to another page or an anchor. They are used for external navigation. 
 
-## Link
+**Best practices**
 
-Description...
+- Use TextButton component if the action is related to internal navigation.
+- Clicking on a Link opens a new tab.
+- Links should always be used with an icon. If it is an external link, use the ExternalLink icon accordingly.
+- Links text should describe an action and start with a verb.
```

Comment:
they should describe the destination of the link and not start with a verb actually.

For instance, saying "Go back" does not provide detail about the navigation. What if the person was accessing the page using the URL? Does go back bring them to google?

My suggestion would be "Homepage", which is meaningful in more situations

Reply:
That's another topic we could discuss later on 😅 I'll remove this sentence